src/v/raft/state_machine.cc

```diff
@@ -168,4 +168,10 @@ ss::future<result<model::offset>> state_machine::insert_linearizable_barrier(
       });
 }
 
+ss::future<model::offset> state_machine::bootstrap_committed_offset() {
+    return _raft->events()
+      .wait(model::offset(0), model::no_timeout, _as)
```

Comment:
nit: may be good to add a log line about why we use a 0 here and what this returns after raft bootstraps.